### ISSUE TITLE
Edit profile focuses on Delegates/Fundraisers, Giver is completely ignored

### ISSUE THREAD

**Issue by vojtechsimetka** [Giveth/giveth-dapp#110]
Text on 'edit your profile': why does the text only talk to me as if I'm a delegate or a fundraiser, not a Giver? @krrisis offered help with text.

**Comment by satyavh** [comment]
That's an extremely valid point. We probably should have different signup flows for givers and makers. As a giver you don't really need to make a profile. But in the current signup flow we have no idea whether you are a giver or a maker.

What would be the quick fix here?

**Comment by vsimetka** [comment]
Just rephrasing the text for now should do. I will try and schedule a session with @krrisis where we review all the text and edit it.

**Comment by krrisis** [comment]
On the homepage it should first give you the choice between 'Are you a Giver or a  you a Maker?', but that is stated in another issue, this one is just about the profile text.

**Comment by satyavh** [comment]
@krrisis Can you rephrase the text and post it here? Then I'll implement it. Thanks

**Comment by krrisis** [comment]
you could temporarily add 'As a Giver you can decide to stay anonymous but if you want to stay-up-to date, leave a mail address for contact purposes.' 
But would also prefer to go through the entire flow with vojtech once the giver is implemented in the entire flow & rewrite more thoroughly.

**Comment by horus2021** [comment]
This UI/UX. No priority.
